packages/lib/src/components/BacsDD/components/BacsInput.scss

```diff
@@ -0,0 +1,44 @@
+@import '../../../style';
+.adyen-checkout__bacs{
+
+  &--confirm{
+    position: relative;
+
+    .adyen-checkout-input__inline-validation--valid{
+      display: none
+    }
+  }
+
+  .adyen-checkout__field{
+
+    &--inactive{
+      pointer-events: none;
+    }
+  }
+
+  .adyen-checkout__bacs--edit{
```

Comment:
```suggestion
  .adyen-checkout__bacs--edit {
```

Reply:
Can the linter not be applied to .scss files?